src/commands/serve.js

```diff
@@ -94,17 +101,11 @@ class ServeCommand extends Command {
               await extensionService.createExtensionUUID()
             }
             distPath = `./dist/dc_${manifest.extensionUUID}.umd.min.js`
-            this.logger.info(`Realizando build da extensão...`)
             await extensionService.build(entryPoint, { mode: 'staging' })
-            this.logger.info(
-              `Build extensão ${entryPoint.replace(this.projectRoot, '')} finalizado com sucesso!`
-            )
           }
-          this.logger.info(`Fazendo upload do arquivo ${distPath} para o Quoti...`)
           const fileBuffer = fs.readFileSync(distPath || changedFilePath)
           const extensionCode = fileBuffer.toString()
-
-          extensionService.upload(fileBuffer, this.getUploadFileName(manifest))
+          await extensionService.upload(fileBuffer, this.getUploadFileName(manifest))
```

Comment:
@luizAlbuquerqueJr, A gente não deve usar o await aqui, o qt serve deve ser usado primariamente só com o websocket e esse upload é só um fallback, não queremos que o usuário tenha que esperar o upload pra poder enviar o código pro front pelo socket (que é feito abaixo).

Tenta manter em mente fazer alterações só pertinentes à task na PR, por favor?

```suggestion
          extensionService.upload(fileBuffer, this.getUploadFileName(manifest))
```

Reply:
sabia que tinha algum truque nesse trecho de código. Irei remover o await deixa comigo.

Reply:
You da man 
<img src="https://media4.giphy.com/media/3oEdv6sy3ulljPMGdy/giphy.gif"/>